jp2_pc/Source/Lib/GeomDBase/TerrainTexture.hpp

```diff
@@ -88,6 +89,7 @@ namespace NMultiResolution
 	private:
 		class CPriv;
 		friend class CPriv;
+		friend class CScheduleTerrainTextureItem;
 
```

Comment:
Friend class declaration is needed so that `CScheduleTerrainTextureItem` can access `DoTextureUpdate()` in `Lib/GeomDBase/TerrainTexture.cpp` line 463.